.github/workflows/pytest_e2e_pr_approval.yml

```diff
@@ -0,0 +1,77 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - staging
+    types:
+      - opened
+      - synchronize
```

Comment:
@sevenearths lets avoid running the tests on every PR update. I think we should do it when it is opened and approved

Reply:
Those are there to re-run it when the code has been updated on a pr

Reply:
No, we said that the tests are not going to run the tests on every PR updated, only when opened and approved